src/FeatureExporter.php

```diff
@@ -85,6 +86,8 @@ public function exportDirectory(
 
         if (is_string($tag)) {
             $this->exportAllFeaturesToSingleFile($features, $targetDirectory, $tag, $stylesheet, $reformatHtml);
+        } elseif ($merge) {
+            $this->exportAllFeaturesToSingleFile($features, $targetDirectory, 'index', $stylesheet, $reformatHtml);
```

Comment:
If you want to merge features tagged "index", you can just provide `--tag index` as CLI option. I understand you want to add the CLI option to merge anyway, but I'd rather not "abuse" the tag argument for that. Maybe this can be implemented in a more generic way?

Reply:
What I want is got a single file without having to use tagged features. I pushed a new commit, hope it will be more like as you are thinking of.

(first contribution to github, please be indulgent)